maas-api.yaml

```diff
@@ -1,555 +1,677 @@
+            type: string 
+        - name: embarkationDate
+          in: query
+          schema:
+            type: string
+            format: date-time
+          required: true
+          description: Departure time of service in ISO8601 format.
+        - name: encode
+          in: query
+          schema:
+            type: boolean
+            default: false
+          description: Set to `true` to receive waypoints as encoded polyline.
+        - name: startStopCode
+          in: query
+          schema:
+            type: string
+          description: Stop code of embarkation. First stop to be considered "travelled".
+          required: false
+        - name: endStopCode
+          in: query
+          schema:
+            type: string
+          description: Stop code of disembarkation. Last stop to be considered "travelled".
+          required: false
+      responses:
+        200:
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  shapes:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/ServiceShape'
+                  realTimeStatus:
+                    type: string
+                    description: Indicator for whether real-time information is available for this service. Note that even if this says the service is real-time, this does not mean the `startTime` is real-time.
+                    enum:
+                      - IS_REAL_TIME
+                      - CAPABLE
+                      - INCAPABLE
+                  realTimeVehicle:
+                    # description: The vehicle for this specific service.
+                    $ref: '#/components/schemas/RealTimeVehicle'
+                  realTimeAlternativeVehicle:
+                    description: Alternative vehicles for the same route, but not necessarily, this specific service.
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/RealTimeVehicle'
+                  alerts:
+                    description: Service alerts / disruption information that affect this service.
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/RealTimeAlert'
+                  bicycleAccessible:
+                    type: boolean
+                    description: Can you take a bicycle on this service? Missing when unknown.
+                  wheelchairAccessible:
+                    type: boolean
+                    description: |
+                      If this service is wheelchair accessible or not. If this field is missing, this does not mean that it is not accessible, but rather that it is unknown whether it is accessible. Only if the value is present and is `false`, does it mean that the service is not wheelchair accessible.
+
+  /routes/filter:
```

Comment:
can we just make this `/routes`?

Reply:
Yep, with your other suggestion that also means that `GET /routes` and `POST /routes` have the same response schema. Hooray.